docs/components.rst

```diff
@@ -0,0 +1,76 @@
+How to use components
+=====================
+
+
+So you've seen the new fancy buttons and want to give them a try, but dont know where to start. No problem!
+
+First lets cover the basics. Discord messages can have *components*, such as buttons, dropdowns etc. These components sit in whats called an "action row". An action row can hold 5 components at a time. Your message can have 5 action rows, so thats a total of 25 components!
+
+Sending some components
+_______________________
+
+First we need to create some buttons, lets put them in a list for now. We'll use :meth:`create_button() <discord_slash.utils.manage_components>` to create a green button
+
+.. code-block:: python
+
+    from discord_slash.utils import manage_components
+    from discord_slash.model import ButtonStyle
+
+    buttons = [
+                manage_components.create_button(
+                    style=ButtonStyle.green,
+                    label="A Green Button"
+                )
+              ]
+
+So we have a button, but where do we use it. Let's create an action row with :meth:`create_actionrow() <discord_slash.utils.manage_components>` and put our buttons in it
```

Comment:
Shouldn't it be :func: instead of :meth:?
```suggestion
So we have a button, but where do we use it. Let's create an action row with :func:`create_actionrow() <discord_slash.utils.manage_components>` and put our buttons in it
```